distributions/aws/test/e2e/tests/test_rds_s3.py

```diff
@@ -0,0 +1,444 @@
+
+RDS_S3_KUSTOMIZE_MANIFEST_PATH = "../../examples/rds-s3/"
+RDS_S3_CLOUDFORMATION_TEMPLATE_PATH = "./resources/cloudformation-templates/rds-s3.yaml"
+CUSTOM_RESOURCE_TEMPLATES_FOLDER = "./resources/custom-resource-templates"
+DISABLE_PIPELINE_CACHING_PATCH_FILE = (
+    "./resources/custom-resource-templates/patch-disable-pipeline-caching.yaml"
+)
+
+
+@pytest.fixture(scope="class")
+def kustomize_path():
+    return RDS_S3_KUSTOMIZE_MANIFEST_PATH
+
+
+@pytest.fixture(scope="class")
+def cfn_stack(metadata, cluster, cfn_client, ec2_client, aws_secrets_driver, request):
+    stack_name = rand_name("test-e2e-rds-stack-")
+
+    resp = ec2_client.describe_vpcs(
+        Filters=[
+            {
+                "Name": "tag:alpha.eksctl.io/cluster-name",
+                "Values": [cluster],
+            },
+        ],
+    )
+    vpc_id = resp["Vpcs"][0]["VpcId"]
+
+    resp = ec2_client.describe_subnets(
+        Filters=[
+            {
+                "Name": "tag:alpha.eksctl.io/cluster-name",
+                "Values": [cluster],
+            },
+            {
+                "Name": "tag:aws:cloudformation:logical-id",
+                "Values": ["SubnetPublic*"],
+            },
+        ],
+    )
+    public_subnets = [s["SubnetId"] for s in resp["Subnets"]]
+
+    resp = ec2_client.describe_instances(
+        Filters=[
+            {
+                "Name": "tag:eks:cluster-name",
+                "Values": [cluster],
+            }
+        ],
+    )
+    instances = [i for r in resp["Reservations"] for i in r["Instances"]]
+    security_groups = [i["SecurityGroups"][0]["GroupId"] for i in instances]
+
+    rds_secret = {
+        "username": rand_name("admin"),
+        "password": rand_name("Kubefl0w"),
+    }
+
+    s3_secret = {
+        "accesskey": get_accesskey(request),
+        "secretkey": get_secretkey(request),
+    }
+
+    return create_cloudformation_fixture(
+        metadata=metadata,
+        request=request,
+        cfn_client=cfn_client,
+        template_path=RDS_S3_CLOUDFORMATION_TEMPLATE_PATH,
+        stack_name=stack_name,
+        metadata_key="test_rds_s3_cfn_stack",
+        create_timeout=10 * 60,
+        delete_timeout=10 * 60,
+        params={
+            "VpcId": vpc_id,
+            "Subnets": ",".join(public_subnets),
+            "SecurityGroupId": security_groups[0],
+            "DBUsername": rds_secret[
+                "username"
+            ],  # remove once katib secrets manager added
+            "DBPassword": rds_secret[
+                "password"
+            ],  # remove once katib secrets manager added
+            "RDSSecretString": create_secret_string(rds_secret),
+            "S3SecretString": create_secret_string(s3_secret),
+        },
+    )
+
+
+KFP_MANIFEST_FOLDER = "../../../../apps/pipeline/upstream/env/aws"
+KATIB_MANIFEST_FOLDER = (
+    "../../../../apps/katib/upstream/installs/katib-external-db-with-kubeflow"
+)
+
+# KFP_MINIO_ARTIFACT_SECRET_PATCH_ENV_FILE = (
```

Comment:
Remove all these commented out lines referring to old env files replaces with secrets manager in final revision of pr?

Reply:
right, missed that

thanks!